app/javascript/components/signups/SignUpForm.tsx

```diff
@@ -341,7 +340,29 @@ const SignUp = () => {
                 </select>
               </Form.Item>
 
-              {/* <Divider>Application</Divider> */}
+              <Divider>Availability</Divider>
```

Comment:
This is the meat of the UI change I added. The other changes are whitespace changes